test/integration/I18n_test.rb

```diff
@@ -433,7 +433,7 @@ class I18nTest < ActionDispatch::IntegrationTest
       follow_redirect!
 
       get '/tag/some-tag'
-      assert_select 'a span.d-none', I18n.t('tag.show.wiki_pages')
+      assert_select 'a > span.d-none', I18n.t('tag.show.wiki_pages')
```

Comment:
This is the last remaining assert_select problem, it's now a failure, not an error!

```
==== FAIL["test_should_choose_i18n_for_tag/show", #<Minitest::Reporters::Suite:0x000055fc54390868 @name="I18nTest">, 122.09785266000006]
 test_should_choose_i18n_for_tag/show#I18nTest (122.10s)
        Expected at least 1 element matching "a > span.d-none", found 0..
        Expected 0 to be >= 1.
        test/integration/I18n_test.rb:436:in `block (2 levels) in <class:I18nTest>'
        test/integration/I18n_test.rb:431:in `each'
        test/integration/I18n_test.rb:431:in `block in <class:I18nTest>'
```

we need to add `d-lg-inline` in between those two nests

Reply:
Hmmm. Looks like these actually passed while during integration test phase, but then somehow get run again during system tests, and then fail then??? Strange!

```

ERROR["test_should_choose_i18n_for_tag/show", #<Minitest::Reporters::Suite:0x000056155d689b60 @name="I18nTest">, 155.05812749199993]
 test_should_choose_i18n_for_tag/show#I18nTest (155.06s)
Minitest::UnexpectedError:         ArgumentError: wrong number of arguments (given 3, expected 1)
            test/integration/I18n_test.rb:436:in `block (2 levels) in <class:I18nTest>'
            test/integration/I18n_test.rb:431:in `each'
            test/integration/I18n_test.rb:431:in `block in <class:I18nTest>'

=ERROR["test_should_choose_i18n_for_wiki/show", #<Minitest::Reporters::Suite:0x000056155159ba48 @name="I18nTest">, 207.88893608799992]
 test_should_choose_i18n_for_wiki/show#I18nTest (207.89s)
Minitest::UnexpectedError:         ArgumentError: wrong number of arguments (given 3, expected 1)
            test/integration/I18n_test.rb:360:in `block (2 levels) in <class:I18nTest>'
            test/integration/I18n_test.rb:355:in `each'
            test/integration/I18n_test.rb:355:in `block in <class:I18nTest>'
```

Reply:
aha i had the system test command syntax wrong, fixed it. it had been running all tests twice, i think, on the system test step!